### PR TITLE
Add support for the PHAT table

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
100% untested with an actual PHAT blob. I'll wait to try this on actual hardware before marking it ready for review.

**Comment by superm1** [comment]
> 100% untested with an actual PHAT blob. I'll wait to try this on actual hardware before marking it ready for review.

Presumably whatever prompted this - they should be able to provide a demo blob from a real system?

**Comment by hughsie** [comment]
> they should be able to provide a demo blob from a real system

Ya, Intel are sorting it out.

File: plugins/acpi-phat/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Platform Health Assessment Table
```
Comment (superm1) [inline plugins/acpi-phat/README.md:1]: I think you want to actually link back to the ACPI spec (IE here: https://uefi.org/htmlspecs/ACPI_Spec_6_4_html/05_ACPI_Software_Programming_Model/ACPI_Software_Programming_Model.html#platform-health-assessment-table)

**Comment by hughsie** [comment]
> <offset>0x54b</offset>

I think all of these should probably disappear from the report attribute too.

File: plugins/acpi-phat/README.md (hunk 1 above)
Comment (hughsie) [reply plugins/acpi-phat/README.md:1]: Fixed.

**Comment by hughsie** [comment]
> I think all of these should probably disappear from the report attribute too.

I did this too.

**Pull request merged**